addons/account/models/company.py

```diff
@@ -125,9 +125,9 @@ class ResCompany(models.Model):
     account_sale_receipt_tax_id = fields.Many2one('account.tax', string="Default Sale Receipt Tax", check_company=True)
     account_purchase_receipt_tax_id = fields.Many2one('account.tax', string="Default Purchase Receipt Tax", check_company=True)
     tax_calculation_rounding_method = fields.Selection([
+        ('round_globally', 'Round per Tax Group'),
```

Comment:
I think renaming this is a very bad idea and is completely wrong. To be discussed with @chklop